mongodb.te

```diff
@@ -60,6 +60,16 @@ manage_files_pattern(mongod_t, mongod_tmp_t, mongod_tmp_t)
 manage_sock_files_pattern(mongod_t, mongod_tmp_t, mongod_tmp_t)
 files_tmp_filetrans(mongod_t, mongod_tmp_t, { file dir sock_file })
 
+## In MongoDB 3.4.16, 3.6.6, 4.0.0 and later, mongod reads netstat info
+## from proc and stores it in its diagnostic system (FTDC).
+## See: https://jira.mongodb.org/browse/SERVER-31400
+## This means that we need to adjust the policy so that the mongod
+## process is allowed to open and read /proc/net/netstat, which
+## also typically has symlinks (e.g. /proc/net/snmp).
+allow mongod_t proc_net_t:file read;
```

Comment:
Perhaps this should instead be one of:
allow mongod_t proc_net_t:file { open read };
allow mongod_t proc_net_t:file { getattr open read };

?